src/pytest_mock_resources/container/async_/postgres.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
+
+from sqlalchemy.ext.asyncio import create_async_engine
+
+
+async def _get_sqlalchemy_async_engine(config, database_name, isolation_level=None):
+    URI_TEMPLATE = (
+        "postgresql+asyncpg://{username}:{password}@{host}:{port}/{database}?ssl=disable"
+    )
+    DB_URI = URI_TEMPLATE.format(
+        host=config.host,
+        port=config.port,
+        username=config.username,
+        password=config.password,
+        database=database_name,
+    )
+
+    options = {}
+    if isolation_level:
+        options["isolation_level"] = isolation_level
+
+    # Trigger any asyncpg-based import failures
+    from pytest_mock_resources.compat import asyncpg
+    asyncpg.connect
+
+    engine = create_async_engine(DB_URI, **options)
+
+    async with engine.connect() as conn:
```

Comment:
I see no reason to `connect` here. We should **know** that it's going to work already, so it saves us the need for
1. this needing to be `async`. If we dont connect, it can be sync
2. `get_sqlalchemy_async_engine` doesn't really need to exist, we can just return the engine directly here, and not need a handle on the loop

Reply:
Oops. I was supposed to remove this.